### PR TITLE
Proposal for new issue template for bug reports and new feature requests

### PR DESCRIPTION
**Pull request by guzman-raphael**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+### Problem
+A clear and concise description how this idea has manifested and the context. Elaborate on the need for this feature and/or what could be improved. Ex. I'm always frustrated when [...]
+
+### Requirements
+A clear and concise description of the requirements to satisfy the new feature. Detail what you expect from a successful implementation of the feature. Ex. When using this feature, it should [...]
+
+### Justification
+Provide the key benefits in making this a supported feature. Ex. Adding support for this feature would ensure [...]
+
+### Alternative Considerations
```
Comment (ixcat) [inline .github/ISSUE_TEMPLATE/feature_request.md:21]: in some cases, people might not have this, so optional?

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+## Feature Request
+
+### Problem
+A clear and concise description how this idea has manifested and the context. Elaborate on the need for this feature and/or what could be improved. Ex. I'm always frustrated when [...]
+
+### Requirements
+A clear and concise description of the requirements to satisfy the new feature. Detail what you expect from a successful implementation of the feature. Ex. When using this feature, it should [...]
+
+### Justification
```
Comment (ixcat) [inline .github/ISSUE_TEMPLATE/feature_request.md:18]: In some cases, the problem should be self sufficient to explain, so optional? Though I suppose people could just say this..

**Review by ixcat — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 1 above)
Comment (guzman-raphael) [reply .github/ISSUE_TEMPLATE/feature_request.md:21]: Based on feedback from @dimitri-yatsenko, removed all mention of optional to keep it simple. There is no way to enforce the structure anyway so the template should be treated as a guide/suggestion.

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 2 above)
Comment (guzman-raphael) [reply .github/ISSUE_TEMPLATE/feature_request.md:18]: Similar to above comment.

**Review by dimitri-yatsenko — APPROVED** [review]
Awesome!

**Pull request merged**